packages/components/src/components/action-group/action-group.tsx

```diff
@@ -212,8 +279,16 @@ export class ActionGroup extends LitElement {
 
   override render(): JsxNode {
     return (
-      <div ariaLabel={this.label} class={CSS.container} role="group">
-        <slot />
+      <div
+        ariaLabel={this.label}
+        class={CSS.container}
+        role={
+          this.selectionMode === "multiple" || this.selectionMode === "none"
+            ? "group"
+            : "radiogroup"
```

Comment:
Wouldn't `radiogroup` require actions use a `radio` role? cc @geospatialem

Reply:
Good catch, @jcfranco! 🐟 Yes, agreed the child `action`s should have a role of `"radio"` when the parent `action-group`'s role is set as `"radiogroup"` 💪🏻